src/angular-app/bellows/apps/public/signup/signup-app.component.html

```diff
@@ -42,7 +42,6 @@ <h1>Create an account</h1>
                     </div>
                 </div>
                 <div>
-                    <script src="/dist/zxcvbn/zxcvbn.js"></script>
```

Comment:
Just want to verify that the password strength feature that zxcvbn provides still works properly after removing this.

Reply:
I did test this locally and it still worked.

Reply:
great thanks